2024-05-15 15:25:02,998:__main__:Trying to divide by zero.
Traceback (most recent call last):
  File "/home/karen/alx-backend-user-data/0x00-personal_data/log_sample.py", line 50, in divide
    result = x / y
ZeroDivisionError: division by zero
[TRIAL] 2024-05-15 16:48:26,422:__main__:Trying to divide by zero.
Traceback (most recent call last):
  File "/home/karen/alx-backend-user-data/0x00-personal_data/log_sample.py", line 50, in divide
    result = x / y
ZeroDivisionError: division by zero
